axoupdater/src/errors.rs

```diff
@@ -166,4 +170,26 @@ pub enum AxoupdateError {
     )]
     #[diagnostic(help("This probably isn't your fault; please open an issue at https://github.com/axodotdev/axoupdater!"))]
     CleanupFailed {},
+
+    /// User passed conflicting GitHub API environment variables
+    #[error("Both {ghe_env_var} and {github_env_var} have been set in the environment")]
```

Comment:
do we want to add the installer_download_url to this?

Reply:
I don't think so - we don't use it in the updater.